cloudify_cli/commands/upgrade.py

```diff
@@ -88,26 +73,103 @@ def upgrade(validate_only,
                     task_retries=task_retries,
                     task_retry_interval=task_retry_interval,
                     task_thread_pool_size=task_thread_pool_size)
+        logger.info('Upgrade validation completed successfully')
 
     if not validate_only:
-        logger.info('Starting manager upgrade process...')
-        _put_workflow_state_file(is_upgrade=True)
-
-        logger.info('Executing manager upgrade...')
-        env.execute('install',
-                    task_retries=task_retries,
-                    task_retry_interval=task_retry_interval)
-
-    logger.info('Deactivating manager maintenance-mode...')
-    client.maintenance_mode.deactivate()
+        try:
+            logger.info('Executing Manager upgrade...')
+            env.execute('install',
+                        task_retries=task_retries,
+                        task_retry_interval=task_retry_interval,
+                        task_thread_pool_size=task_thread_pool_size)
+        except Exception as e:
+            msg = "'Failed upgrading Manager. Error: {0}".format(e.message)
```

Comment:
don't use str(e), use e as it is already part of format so it gets called implicitly